### PR TITLE
Add coverage for hostgroup redirection functionality(BZ1402390)

### PR DESCRIPTION
**Pull request by oshtaier**
```
nosetests tests/foreman/ui/test_hostgroup.py -m test_positive_redirection_for_multiple_hosts
.
----------------------------------------------------------------------
Ran 1 test in 86.594s

OK
```

### CODE REVIEW

File: tests/foreman/ui/test_hostgroup.py [hunk 1]
```diff
@@ -190,6 +190,47 @@ def test_positive_create_with_activation_keys(self):
...
     @tier1
+    def test_positive_redirection_for_multiple_hosts(self):
+        """Create new hostgroup with whitespaces in its name for specific
+        organization and location. After that create some hosts using already
+        created hostgroup
+
+        :id: f301ec2f-1e9b-4e34-8c87-4d0b3381d9e1
+
+        :expectedresults: Hostgroup hosts count can be successfully observed
+            in a table and redirection to these hosts works properly
```
Comment (abalakh) [inline tests/foreman/ui/test_hostgroup.py]: Redirection is not tested. Pls add a step to click the link and check name field contains hostname or something similar.

Comment (oshtaier) [inline tests/foreman/ui/test_hostgroup.py]: emm? =)
click:
```
self.hostgroup.click(
    common_locators['table_cell_link'] % (name, 'Hosts'))
```
check:
```
for host_name in host_names:
    self.hosts.wait_until_element(
        locators['host.select_name'] % host_name)
```
That is what was expected to be tested

Comment (oshtaier) [inline tests/foreman/ui/test_hostgroup.py]: redirection to the host table page

Comment (oshtaier) [inline tests/foreman/ui/test_hostgroup.py]: In case of any question - check the video attached to the BZ

Comment (abalakh) [inline tests/foreman/ui/test_hostgroup.py]: Got it now, thanks. Was initially thinking it's some kind of 'Hosts' tab under specific hostgroup, for that case redirection to host page would be a good thing to test (as we often have bugs in such links).

Comment (oshtaier) [inline tests/foreman/ui/test_hostgroup.py]: Yeah, but there is no host tab in hostgroup entity (which sounds not so logical for me. Host Group should be treated like group of hosts, but not for us, not for us =)). Redirection can be performed from table only as far as I see. Anyway thanks for the comment

**Pull request merged**